src/ecs/mod.rs

```diff
@@ -1,5 +1,6 @@
 //! `amethyst` engine built-in types for `specs`.
 
+pub use shred::*;
```

Comment:
We should not use two glob imports here. This could easily cause breakage.

Reply:
Even one global import from another crate could break compilation.
Maybe we shouldn't import globally at all?
Replacing all `ecs::{foo, bar, baz}` with `specs::{foo, bar, baz}` doesn't look bad.